### ISSUE TITLE
Compile Error: Declaration of Translation\Bundle\Twig\Visitor\NormalizingNodeVisitor::doEnterNode(Twig_Node $node, Twig_Environment $env) must be compatible with Twig\NodeVisitor\AbstractNodeVisitor::doEnterNode(Twig\Node\Node $node, Twig\Environment $env) 

### ISSUE THREAD

**Issue by grekpg** [php-translation/symfony-bundle#289]
```
 16:00:36 CRITICAL  [php] Fatal Compile Error: Declaration of Translation\Bundle\Twig\Visitor\NormalizingNodeVisitor::doEnterNode(Twig_Node $node, Twig_Environment $env) must be compatible with Twig\NodeVisitor\AbstractNodeVisitor::doEnterNode(Twig\Node\Node $node, Twig\Environment $env) ["exception" => Symfony\Component\Debug\Exception\FatalErrorException { …}]
!!  
!!  In NormalizingNodeVisitor.php line 59:
!!                                                                                 
!!    Compile Error: Declaration of Translation\Bundle\Twig\Visitor\NormalizingNo  
!!    deVisitor::doEnterNode(Twig_Node $node, Twig_Environment $env) must be comp  
!!    atible with Twig\NodeVisitor\AbstractNodeVisitor::doEnterNode(Twig\Node\Nod  
!!    e $node, Twig\Environment $env)                             
```

  "require": {

....
        "php-translation/symfony-bundle": "^0.8.1",

        "sensio/framework-extra-bundle": "^5.2",
        "sg/datatablesbundle": "^1.0",

        "sonata-project/admin-bundle": "^3.40",
        "sonata-project/user-bundle": "^4.2",
        "sonata-project/doctrine-orm-admin-bundle": "^3.6",
        "sonata-project/translation-bundle": "^2.4",

        "sonata-project/page-bundle": "^3.9",

        "spotonlive/php-google-my-business": "^4.3",
        "stof/doctrine-extensions-bundle": "^1.3",
        "symfony/acl-bundle": "^1.0",
        "symfony/asset": "^4.1",
        "symfony/console": "^4.1",
        "symfony/expression-language": "^4.1",
        "symfony/flex": "^v1.0.83",
        "symfony/form": "^4.1",
        "symfony/framework-bundle": "^4.1",
        "symfony/lts": "^4@dev",
        "symfony/monolog-bundle": "^3.1",
        "symfony/orm-pack": "*",
        "symfony/process": "^4.1",
        "symfony/security-bundle": "^4.1",
        "symfony/serializer-pack": "*",
        "symfony/swiftmailer-bundle": "^3.1",
        "symfony/twig-bundle": "^v4.2.4",
        "symfony/validator": "^4.1",
        "symfony/web-link": "^4.1",
        "symfony/webpack-encore-pack": "^1.0",
        "symfony/yaml": "^4.1",
        "tbbc/money-bundle": "dev-master",
        "white-october/pagerfanta-bundle": "^1.2"

**Comment by remyvanlerberghe** [comment]
This bug has nothing to do with this bundle. It has been introduced in Twig's 2.7.0 version. A patch has been released so this issue should be closed.

**Comment by Kocal** [comment]
The same issue was opened (#288) but closed because it has been fixed with Twig 2.7.1 and 1.38.1.

**Comment by odolbeau** [comment]
Looks like it have been solved, thanks guys! :+1:
